src/server/library/core_functions/mg_contracts.py

```diff
@@ -1274,6 +1274,23 @@ def tensorflow_micro():
 tensorflow_micro_contracts = {"input_contract": [], "output_contract": []}
 
 
+def neural_network():
+    """
+    The Neural Network uses Tensorflow Lite for Microcontrollers, an inference engine
+    from Google optimized run machine learning models on embedded devices.
```

Comment:
update this text to be more generic. This is strictly for training. The inference engine can be deployed to NNoM or TFliteMicro